src/views/Stack/Card.tsx

```diff
@@ -544,14 +550,18 @@ export default class Card extends React.Component<Props> {
           this.gestureUntraversed,
           I18nManager.isRTL ? MINUS_ONE_NODE : TRUE_NODE
         ),
-        this.gestureUntraversed
+        multiply(this.gestureUntraversed, this.verticalGestureDirection)
       )
     ),
     set(
       this.velocity,
-      multiply(
-        this.velocityUntraversed,
-        I18nManager.isRTL ? MINUS_ONE_NODE : TRUE_NODE
+      cond(
+        eq(this.direction, DIRECTION_HORIZONTAL),
+        multiply(
+          this.gestureUntraversed,
```

Comment:
Shouldn't this be velocity?

```suggestion
          this.velocityUntraversed,
```